test/boulder-config.json

```diff
@@ -110,7 +110,8 @@
           "expiry": "8760h"
         }
       }
-    }
+    },
+    "maxGoroutines": 0
```

Comment:
I know we talked before I left but I realized that this should probably be maxRPCConcurrentRequests which is the real thing we're doing here.